website/source/docs/templates/engine.html.md

```diff
@@ -74,15 +74,35 @@ Here is a full list of the available functions for reference.
     }
     ```
 
-    Valid variables to request are: "ID", "Host",
-    "Port", "User", "Password", "ConnType",
-    "PackerRunUUID", "PackerHTTPAddr", "SSHPublicKey", and "SSHPrivateKey".
-    Depending on which communicator you are using, some of these values may be
-    empty -- for example, the public and private keys are unique to the SSH
-    communicator. InstanceID represents the vm being provisioned. For example,
-    in Amazon it is the instance id; in digitalocean, it is the droplet id; in
-    Vmware, it is the vm name.
-
+    Valid variables to request are: 
```

Comment:
@SwampDragons can you check if the descriptions for the variables are correct, please? 🙂

Reply:
Done!